lib/models/key.coffee

```diff
@@ -126,8 +128,14 @@ exports.create = (title, key, callback) ->
 	# Avoid ugly whitespaces
 	key = key.trim()
 
-	return pine.post
-		resource: 'user__has__public_key'
-		body: { title, key }
-	.get('id')
+	Promise.props
+		userId: auth.getUserId()
```

Comment:
Since we're only getting the userId, this should be done with a `.then()` instead of `.props`.